proxy/src/control/destination/mod.rs

```diff
@@ -113,6 +128,7 @@ impl Resolver {
 
         Resolution {
             update_rx,
+            receiver_gone: receiver_gone,
```

Comment:
This should just be `receiver_gone` not `receiver_gone: receiver_gone`. (I think the compiler will warn about this; I am surprised the code still builds and I'm guessing we need to `deny(...)` whatever warning that is.)